exercises/square-root/metadata.yml

```diff
@@ -0,0 +1,4 @@
+---
+blurb: "Given a natural radicand, return its square root."
+source: ""
```

Comment:
Cite yourself if it is your own original work.  Got a URL or blog etc?

Reply:
Original work, but I do not have any site. Will add my GitHub profile if acceptable?